sdk/src/main/resources/account/rpc/api/eth_openrpc.json

```diff
@@ -357,7 +343,7 @@
 					"name": "Block",
 					"required": true,
 					"schema": {
-						"title": "Block number or tag",
+						"title": "Block number, block hash or tag",
```

Comment:
update the description, remove `tag`
check other places

Reply:
Done